autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -213,6 +217,12 @@ def _invoke_assistant(
                 role=message["role"],
             )
 
+        self.client.create(
```

Comment:
I don't really understand the log related implement, but I would like to know if it will call openai completion function?

Reply:
.create() calls the log_chat_completion() which is responsible only for logging data into sqlite DB.

its the self._get_run_response() which makes openai chat completions call

Reply:
for the `GPTAssistantAgent` we use the `run` to get the responses from the assistant. i think calling the openai completion doesn't add up to the actual assistant logs.